website/content/contribute/module/index.md

```diff
@@ -0,0 +1,199 @@
+---
+title: Contributing a Module
+date: 2020-11-28T22:10:51+02:00
+draft: false
+weight: 1
+---
+
+This page describes what exactly must be provided when contributing a module, and how to make the control plane aware of the new module so that it can be used.
+
+
+
+Steps for creating a module:
+1. Implement the logic of the module you are contributing - either directly in the [Module Workload](#module-workload) or in an external component.  If the logic is in an external component, then the module workload should act as a client - i.e. receiving paramaters from the control plane and passing them to the external component.
+2. [Create the Module Helm Chart](#module-helm-chart) that will be used by the control plane to deploy the module workload, update it, and delete it as necessary.
```

Comment:
After creating the module Helm chart, it should also be deployed (pushed to a registry).

Reply:
updated